pkg/volume/rbd/rbd_util.go

```diff
@@ -142,8 +151,9 @@ func (util *RBDUtil) rbdLock(b rbdMounter, lock bool) error {
 		// for fencing, check if lock already held for this host
 		// this edge case happens if host crashes in the middle of acquiring lock and mounting rbd
 		// for defencing, get the locker name, something like "client.1234"
-		cmd, err = b.plugin.execCommand("rbd",
```

Comment:
There is `append` in the middle of `plugin.execCommand` arguments. 

It looks ugly to me to have `append([]string{"rbd",<other args>}, secret_opt...)...)` as `b.exec.Run` arguments. Two `...` in one statement look quite confusing to me.